[MTM-46156] Add-requirements-for-managing-tenants (#1554)
SoftwareAG/c8y-docs: content/enterprise-tenant/managing-tenants-bundle/managing-subtenants.md

    To create a subtenant click **Create tenant** at the top right."
---

{{< c8y-admon-req >}}
APPLICATION ACCESS:

> The user must have access to the Administration application of {{< enterprise-tenant >}}

Reviewer: ```suggestion
The user must have access to the Administration application of an {{< enterprise-tenant >}}.
```

> To be able to use the tenant functionality, your user must have the appropriate permissions. See [To add a global role](/standard-tenant/managing-permissions/#to-add-a-global-role) for information on editing permissions. Since editing tenants is a sensitive operation, permissions for editing tenants are more granular:

Reviewer 2: This paragraph is about permissions, should it then go under "ROLES & PERMISSIONS` header below?

ROLES & PERMISSIONS:

> The user must have one or more of "Tenant management" permissions:

Reviewer: ```suggestion
The user must have at least one permission for the permission type "Tenant management":
```

* To view all tenants: READ permission.
* To create tenants: CREATE permission.

> * To edit tenants (incl. subscriptions) and suspend or activate them: UPDATE permission.

Reviewer: ```suggestion
* To edit tenants (including subscriptions) and suspend or activate them: UPDATE permission.
```

> * To create and edit existing tenants: ADMIN permission.

Reviewer 2: Drop "existing". Above we have "edit tenants" which also refers to existing ones, right? So probably no need to specially  emphasize it here.
ADMIN permission also covers suspending and activating tenants, right? As it's written now, it suggests that editing and suspending/activation are separate operations, doesn't it?

Author: extended description for admin permissions.

{{< /c8y-admon-req >}}

> Using the {{< enterprise-tenant >}} of {{< product-c8y-iot >}}, you can make use of the tenants functionality which allows you to create and manage subtenants.

Reviewer: This sentence should go at the very top as an introductory sentence.

Using the {{< enterprise-tenant >}} of {{< product-c8y-iot >}}, you can make use of the tenants functionality which allows you to create and manage subtenants.

{{< c8y-admon-req >}}
APPLICATION ACCESS:

The user must have access to the Administration application of an {{< enterprise-tenant >}}.

ROLES & PERMISSIONS:

> To be able to use the tenant functionality, your user must have the appropriate permissions. See [To add a global role](/standard-tenant/managing-permissions/#to-add-a-global-role) for information on editing permissions. Since editing tenants is a sensitive operation, permissions for editing tenants are more granular.

Reviewer 2: Yes, we describe the requirements here, but not how to set them in UI. The link goes to the section where it is described how to grant permissions to a group, right @Airdroplet?

> To be able to use the tenant functionality, your user must have the appropriate permissions. See [To add a global role](/standard-tenant/managing-permissions/#to-add-a-global-role) for information on editing permissions. Since editing tenants is a sensitive operation, permissions for editing tenants are more granular.

Reviewer: > Yes, we describe the requirements here, but not how to set them in UI. The link goes to the section where it is described how to grant permissions to a group, right @Airdroplet?

But that would mean that we generally have to add the sentence "See [To add a global role](/standard-tenant/managing-permissions/#to-add-a-global-role) for information on editing permissions. " to all requirement sections, wouldn´t it? We do not mention it in other places how to set the requirements.

> To be able to use the tenant functionality, your user must have the appropriate permissions. See [To add a global role](/standard-tenant/managing-permissions/#to-add-a-global-role) for information on editing permissions. Since editing tenants is a sensitive operation, permissions for editing tenants are more granular.

Reviewer 2: right, that makes sense, let's drop it then :smile:

The user must have at least one permission for the permission type "Tenant management":
* To view all tenants: READ permission.
* To create tenants: CREATE permission.
* To edit tenants (including subscriptions) and suspend or activate them: UPDATE permission.

> * To create, edit (including subscriptions), suspend or activate tenants: ADMIN permission.

Reviewer 2: Should we break the format by writing this point in a different style than others?

> * To create, edit (including subscriptions), suspend or activate tenants: ADMIN permission.

Reviewer: I would leave it as is, I just suggest to rephrase it since it sounds confusing to simply list it in the same style even if it is correct.

> * To create, edit (including subscriptions), suspend or activate tenants: ADMIN permission.

Reviewer 2: If I see it right, the difference is that ADMIN additionally allows for creation. Listing it in the same style would allow to quickly spot the difference, if we write it in a different style, it's harder to spot right away. Maybe we could write something like: "same as above, plus tenant creation: ADMIN permission."?

> * To create, edit (including subscriptions), suspend or activate tenants: ADMIN permission.

Author: rephrase done, please check :)
